source/uwp/winui3/Renderer/idl/AdaptiveCards.Rendering.Winui3.idl

```diff
@@ -0,0 +1,668 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace AdaptiveCards
+{
+    namespace Rendering
+    {
+        namespace Winui3
```

Comment:
Just to verify, is this namespace name consistent with other usages of the Winui3 namespace, for example, comparing against the one defined in their repository or how the headers capitalize UI. This could be dismissible if using lowercase allows for less namespace conflicts to exist

Reply:
"WinUI" (with whatever capitalization) actually isn't used a lot in that repository, since the DLLs produced are in the Microsoft.UI.Xaml namespace and don't contain "winui" at all. Although when it is used, I suppose "WinUI" is more common than "Winui". If we want to make this change, I think we would want to do it in the ObjectModel as well.

Reply:
https://github.com/microsoft/AdaptiveCards/pull/8255